app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
```

Comment:
Nepřipadá mi vůbec DRY, že se v každé harvester třídě opakuje kód:

```ruby
class ApiError < StandardError; end

attr_reader :min_id, :downloader

def self.run(min_id: 0, downloader: HarvesterUtils::Downloader)
  new(min_id, downloader).run
end

def initialize(min_id, downloader)
  @min_id = min_id
  @downloader = downloader
end
```

Lepší by IMHO bylo vytvořit třídu, ze které by tyto harvester třídy dědily. Instance metoda run by vypadala takto:
```ruby
class NotImplementedError < StandardError; end

def run
  raise NotImplementedError.new('You must implement method "run"')
end
```